Formula/tldr.rb

```diff
@@ -0,0 +1,19 @@
+class Tldr < Formula
+  desc "Simplified and community-driven man pages"
+  homepage "https://tldr-pages.github.io"
+  url "https://github.com/tldr-pages/tldr-cpp-client/archive/v1.3.0.tar.gz"
+  sha256 "6210ece3f5d8f8e55b404e2f6c84be50bfdde9f0d194a271bce751a3ed6141be"
+  head "https://github.com/tldr-pages/tldr-cpp-client.git"
+
+  depends_on "pkg-config" => :build
+  depends_on "libzip"
+
+  def install
+    system "make"
```

Comment:
I took a look at `Makefile` and the `install` target already depends on `all`, so this line is redundant. I'll push a fix.